.prettierrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
```

Comment:
While I'm happy to add prettier and linting rules to this repo, ideally it would be in a separate PR (makes everything easier to review)

Reply:
yeah right, I added this to my branch because my IDE was changing way to much, I think I will integrate this in the .gitignore